### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by Meme-Eater** [Monika-After-Story/MonikaModDev#8886] [appreciation]
I just wanted to thank you guys for the great work in the mod. I remember when MAS just started, and see how much it has evolved. It's really great! 

Thanks.

**Comment by MetalBrother** [comment]
Pretty much the same, we all started without affection, any costumes or other changes, without compliments and even this was enough, but now this game evolved so much and it became a lot more enjoyable, thanks for great work!

**Comment by ghost** [comment]
i got to say i've never seen such good mod in my life thx whoever is developping MAS

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
